alembic/versions/1b6e419ea1a6_add_threshold1_to_existing_prefs.py

```diff
@@ -39,10 +39,10 @@ def upgrade():
     valid_paths = fmn.lib.load_rules(root='fmn.rules')
 
     filters = session.query(fmn.lib.models.Filter).filter_by(name=target).all()
-    print "Found %i filters" % len(filters)
+    print("Found %i filters" % len(filters))
```

Comment:
I used 2to3 and it very unkindly does not add from __future__ imports. I've fixed that.